### PR TITLE
[bitnami/kubernetes-event-exporter] feat: configure probes

### PR DESCRIPTION
**Pull request by Farfaday**
That follows/reopens https://github.com/bitnami/charts/pull/21093 with the merge conflict fixed.

### Description of the change

Add Probes with default values. Since  https://github.com/resmoio/kubernetes-event-exporter/pull/82 endpoints have been able to do that.

### Benefits

Probes enabled by defaults :)

### Possible drawbacks

None that I can think of yet.

### Applicable issues

None

### Additional information

None

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/kubernetes-event-exporter/Chart.yaml [hunk 1]
```diff
@@ -31,4 +31,4 @@ maintainers:
 name: kubernetes-event-exporter
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kubernetes-event-exporter
-version: 2.9.3
+version: 2.9.4
```
Comment (joancafom) [inline bitnami/kubernetes-event-exporter/Chart.yaml:34]: This should be a minor bump, we are adding a new feature (default configurable probes)

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! Left you a small comment for you to apply :)

File: bitnami/kubernetes-event-exporter/Chart.yaml (hunk 1 above)
Comment (Farfaday) [reply bitnami/kubernetes-event-exporter/Chart.yaml]: ok! Changed to 2.10.0 :)

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your changes 🚀

**Pull request merged**